src/Files.App/Utils/Logger/SentryLogger.cs

```diff
@@ -23,15 +23,19 @@ public bool IsEnabled(LogLevel logLevel)
 		public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception? exception, Func<TState, Exception?, string> formatter)
 		{
 			// Unhandled exceptions are captured in AppLifecycleHelper.HandleAppUnhandledException
-			if (exception is null || exception.Data[Mechanism.HandledKey] is false)
+			if
+			(
+				exception is null ||
+				exception.Data[Mechanism.HandledKey] is false ||
+				logLevel == LogLevel.Information
+			)
 				return;
 
 			var generalSettingsService = Ioc.Default.GetRequiredService<IGeneralSettingsService>();
 
 			var level = logLevel switch
 			{
 				LogLevel.Debug => SentryLevel.Debug,
-				LogLevel.Information => SentryLevel.Info,
```

Comment:
I don't think this needs to be deleted in case you change the level of filtering.

Reply:
It doesn't make much difference so I added it back.